src/meta/src/stream/graph/stream_graph.rs

```diff
@@ -581,6 +582,7 @@ where
                     fields: chain_node.upstream_fields.clone(),
                     operator_id: merge_node.operator_id,
                     identity: "MergeExecutor".to_string(),
+                    append_only: stream_node.append_only,
```

Comment:
Can we determine `append_only` of upstream node solely from the `StreamTableScan`'s `append_only` property?

Reply:
I saw that `append_only` of `StreamTableScan` is always `false`... So I think we can follow the `append_only` property given by frontend for now. If there's issue after we implement the full append-only inference, let's fix it later.

Reply:
In fact I think the information should be in the catalog of the mv...